robyn/__main__.py

```diff
@@ -33,6 +34,39 @@ def index():
             """
         )
 
+    # DockerFile configuration
+    if docker == "yes":
```

Comment:
create a sanitization function here, which will take in the value of the variable and check for the `Y/N` value